### DISCUSSION TITLE
Export sql result  to excel feature

### DISCUSSION THREAD

**Discussion by hoanganf** [dbeaver/dbeaver#15324] [Q&A]
** first thank you for provide the amazing free sql tool, im using it in my work 
**Is your feature request related to a problem? Please describe.**
if you add export sql result to excel file feature, i think the app will are best choice for sql tool

**Describe the solution you'd like**
when i query the sql, and the result can be export to excel file (not csv), because if using csv file we can not resolve the complicate column which has comma in the value (like 'xx,yy')

**Comment by LonwoLonwo** [comment]
Hello @hoanganf 

We already have this feature as an extra extension for DBeaver: https://github.com/dbeaver/dbeaver/wiki/Optional-extensions#office-integration

Also, our commercial product includes this extension from the start: https://dbeaver.com/
